docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -357,7 +357,7 @@ By default, the `ServiceRegistry` implementation will auto-register the running
 
 ==== Service Registry Actuator Endpoint
 
-A `/service-registry` actuator endpoint is provided by Commons. This endpoint relys on a `Registration` bean in the Spring Application Context. Calling `/service-registry/instance-status` via a GET will return the status of the `Registration`. A POST to the same endpoint with a `String` body will change the status of the current `Registration` to the new value. Please see the documentation of the `ServiceRegistry` implementation you are using for the allowed values for updating the status and the values retured for the status.
+A `/service-registry` actuator endpoint is provided by Commons. This endpoint relies on a `Registration` bean in the Spring Application Context. Calling `/service-registry` via a GET will return the status of the `Registration`. A POST to the same endpoint with a JSON body will change the status of the current `Registration` to the new value. The JSON body has to include both the `status` and `overridenStatus` field with the preferred value. Please see the documentation of the `ServiceRegistry` implementation you are using for the allowed values for updating the status and the values returned for the status. For instance, Eureka's supported statuses are `UP`, `DOWN`, `OUT_OF_SERVICE` and `UNKNOWN`.
```

Comment:
I don't think this is true:
> body has to include both the `status` and `overridenStatus`

Looking at the code, it should only take `status`.

Reply:
Right, didn't test that combination. Will change this evening!